games-action/powermanga/patches/powermanga-0.93.1.patchset

```diff
@@ -0,0 +1,72 @@
+From 40817bc1c231d878d803df2bb1b487785b946824 Mon Sep 17 00:00:00 2001
+From: Abdur-Raheem Idowu <abdurraheemidowu@gmail.com>
+Date: Thu, 26 Dec 2019 11:56:43 +0000
+Subject: Correcting install and score directories
+
+
+diff --git a/src/Makefile.am b/src/Makefile.am
+index 0327a54..d9aceae 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -1,6 +1,6 @@
+
+-gamesdir = $(prefix)/games
+-scoredir = /var/games/powermanga
++gamesdir = $(prefix)/apps/Powermanga
++scoredir = $(prefix)/apps/Powermanga/.self/settings
```

Comment:
Not quite right, it's not .self/settings, well in the build it looks that way, it shouldn't be in $prefix/apps/Powermanga also, that part is read-only

Reply:
Do you mind sharing how you did it with me? I've searched all around, but I haven't found how how I can access an apps settings through /system/.

Reply:
Is it by any chance /system/package-links ?

Reply:
No, you need to link it to "settings" (with a prefix)

Reply:
Check /boot/system/settings ...

Reply:
I don't find anything that has to do with Powermanga in there, only bashrc, gitconfig etc. Is this where application settings should be?

Reply:
There isn't anything related to powermanga there because it hasn't been installed there (yet), some apps save their settings there (or /boot/home/config/settings, but atm system is prefered)

Reply:
Didn't have any idea about this, and couldn't find it online, just saw that most apps installed in /boot/home/config/settings and that is why I originally used that.
I hope you understand.